ansible-ipi-install/README.md

```diff
@@ -335,6 +335,9 @@ dnsvip=""
 # A copy of your pullsecret from https://cloud.redhat.com/openshift/install/metal/user-provisioned
 pullsecret=""
 
+# An optional list of clock servers to be used in chrony by the masters and workers
+#clock_servers=["pool.ntp.org","clock.redhat.com"]
+
```

Comment:
Could you rebase off the latest master branch and place this piece under 
```
# (Optional) Enable IPv6 addressing instead of IPv4 addressing
#ipv6_enabled=True
```
I want to keep all vars under this banner 
```
######################################
# Vars regarding install-config.yaml #
######################################
```
related to install-config.yaml

And thanks again for the PR looks great!